Tests/iaas/flavor-naming/flavor-names-openstack.py

```diff
@@ -13,16 +13,17 @@
 provided by openstack, such as the number of vCPUs and memory.
```

Comment:
The module level comment is now outdated:

* the script doesn't use `flavor-name-check.py` anymore
* https://github.com/SovereignCloudStack/Docs/Design-Docs/tools is a dead link (but that's not caused by this PR)